config.yaml

```diff
@@ -100,3 +100,28 @@ options:
         beyond the pool size when the pool is exhausted.
     default: 10
     type: int
+  google-client-id:
+    description: |
+      Client id from Google Oauth setup
+    type: string
+  google-client-secret:
+    description: |
+      Client password from Google Oauth setup
+    type: string
+  oauth-domain:
+    description: |
+      The domain for which to allow google authentication ie. canonical.com
+    type: string
+  oauth-admin-email:
+    description: |
+      Email(s) to be given an Admin role on initialization.
+      Either a single email or a comma separated list.
+      eg. "example.email@company.com, example.email2@company.com"
+    type: string
+    default: admin@superset.com
+  self-registration-role:
+    description: |
+      The default role to be provided to users that self-register via Oauth.
+      One of `Admin`, `Alpha`, `Gamma`, `Public`, `sql_lab`
```

Comment:
Personal curiosity, what are the `Alpha` and `Gamma` roles? Where do they come from?

Reply:
They are pre-defined by Superset, you can see the breakdown of permissions here: https://github.com/apache/superset/blob/master/RESOURCES/STANDARD_ROLES.md